### DISCUSSION TITLE
How to use katex plugins?

### DISCUSSION THREAD

**Discussion by DrakoPOD** [vuejs/vitepress#1248] [Q&A] [answered]
Trying to figure out how to use katex, but it does't render properly.

What config I need to use?

**Comment by brc-dd** [comment, marked as answer]
https://github.com/vuejs/vitepress/issues/529

**Reply by DrakoPOD** [reply]
I already saw that, mathjax doesn't render properly, equations aren't centered and inline equations displays in a new line instead. Katex, just simply doesn't work.

**Reply by brc-dd** [reply]
@DrakoPOD You probably haven't added styles: https://github.com/brc-dd/vitepress-mathjax3/blob/main/.vitepress/theme/styles/mathjax3.css

**Reply by DrakoPOD** [reply]
Yes, forgot the `index.js` on themes forlder.
